RecoTracker/LSTCore/src/alpaka/LSTEvent.dev.cc

```diff
@@ -600,22 +583,18 @@ void LSTEvent::createTrackCandidates(bool no_pls_dupclean, bool tc_pls_triplets)
   alpaka::wait(queue_);  // wait to get the value before using
   auto const nEligibleModules = *nEligibleModules_buf_h.data();
 
-  Vec3D const threadsPerBlockRemoveDupQuints{1, 16, 32};
-  Vec3D const blocksPerGridRemoveDupQuints{1, std::max(nEligibleModules / 16, 1), std::max(nEligibleModules / 32, 1)};
-  WorkDiv3D const removeDupQuintupletsBeforeTC_workDiv =
-      createWorkDiv(blocksPerGridRemoveDupQuints, threadsPerBlockRemoveDupQuints, elementsPerThread);
+  auto const removeDupQuintupletsBeforeTC_workDiv = cms::alpakatools::make_workdiv<Acc2D>(
+      {std::max(nEligibleModules / 16, 1), std::max(nEligibleModules / 32, 1)}, {16, 32});
 
-  alpaka::exec<Acc3D>(queue_,
+  alpaka::exec<Acc2D>(queue_,
                       removeDupQuintupletsBeforeTC_workDiv,
                       RemoveDupQuintupletsBeforeTC{},
                       quintupletsDC_->view<QuintupletsSoA>(),
                       quintupletsDC_->view<QuintupletsOccupancySoA>(),
                       rangesDC_->const_view());
 
-  Vec3D const threadsPerBlock_crossCleanT5{32, 1, 32};
-  Vec3D const blocksPerGrid_crossCleanT5{(13296 / 32) + 1, 1, max_blocks};
-  WorkDiv3D const crossCleanT5_workDiv =
-      createWorkDiv(blocksPerGrid_crossCleanT5, threadsPerBlock_crossCleanT5, elementsPerThread);
+  auto const crossCleanT5_workDiv =
+      cms::alpakatools::make_workdiv<Acc3D>({(13296 / 32) + 1, 1, max_blocks}, {32, 1, 32});
```

Comment:
not sure if it's worth to comment here that 13296 is nLowerModules or set it to that value explicitly.

Reply:
Yeah, it's definitely better to use that explicitly